src/user/announcements.rst

```diff
@@ -8,6 +8,20 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2023
 ----
 
+2023-08-24: Bumping Minimum MacOS version to 10.13
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We will bump the minimum MacOS version from 10.9 (released in Oct. 2013, end-of-life
+since Dec. 2016) to 10.13 (released Sept. 2017, end-of-life since Dec. 2020).
+Several core packages in the ecosystem either now, or will soon, require at least 10.13.
+These packages include the C++ standard library for OSX itself, ``libcxx``,
```

Comment:
I've committed your suggestion @beckermr, and I'm fine if we leave it at that. I do/did find it relevant to note the relation between the SDK and how we ship libcxx, because I'm pretty sure that this aspect is understood only by very few people; I suspect that most users have no idea we support such old MacOS, or consider it black magic (regulars in conda-forge will have seen `_LIBCPP_DISABLE_AVAILABILITY` but that doesn't mean they understand how this works; at least I didn't for a long time).

Something like the following
```suggestion
The main reason we managed to support 10.9 this long at all, is that conda-forge is able to ship an
up-to-date C++ standard library for OSX, ``libcxx``, in its environments, replacing the old one
in the SDK.

However, several core packages in the ecosystem either now, or will soon, require at least 10.13,
in a way that we cannot be circumvent. These packages include ``libcxx``,
```

Reply:
Sure go for it!